js/web/lib/wasm/wasm-training-core-impl.ts

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {InferenceSession} from 'onnxruntime-common';
+
+import {SerializableModeldata, SerializableSessionMetadata} from './proxy-messages';
+import {setSessionOptions} from './session-options';
+import {getInstance} from './wasm-factory';
+import {checkLastError} from './wasm-utils';
+
+const NO_TRAIN_FUNCS_MSG = 'Built without training APIs enabled. ' +
```

Comment:
Please improve the message. You should add how to or where to get the package built with training apis. Example add the build options they need to use to build the native package or paste a link from where users can download a prebuilt package with training apis.